clients/context-registry/src/ContextRegistryClient.ts

```diff
@@ -115,122 +106,116 @@ export class Project extends CommonAssetProjectContext {
 }
 
 /** An iTwin context of type asset. Assets represent a large scale item that is owned and/or operated by organization, such as buildings, highways and so on.
+ * Deprecated
  * @beta
  */
 @ECJsonTypeMap.classToJson("wsg", "CONNECTEDContext.Asset", { schemaPropertyName: "schemaName", classPropertyName: "className" })
-export class Asset extends CommonAssetProjectContext {
+class Asset extends HiddenContext {
   @ECJsonTypeMap.propertyToJson("wsg", "properties.AssetType")
   public assetType?: string;
 }
 
-/** Options to request iTwin contexts.
- * @beta
- */
-export interface ContextRegistryRequestQueryOptions extends RequestQueryOptions {
-  /** Set to true to request the most recently used projects */
-  isMRU?: boolean;
-
-  /** Set to true to request the favorite projects */
-  isFavorite?: boolean;
-}
-
 /** Client API to access the context registry services.
  * @beta
  */
-export class ContextRegistryClient extends WsgClient {
+export class ITwinAccessClient extends WsgClient implements ITwinAccess {
```

Comment:
If we are changing the client class shouldn't we also change the file and package name?

Reply:
This client is going to go away entirely before 3.0 is released and all current references in tests will use a mock going forward. Then the test-apps will use the new implementation once it's available.

As a transition plan, while we get everything into place, we're providing a wrapper around the current client to keep everything working as is to allow this to happen in stages.

Reply:
ok